src/projetos/SugestaoPage.tsx

```
@@ -150,6 +139,30 @@ export function SugestaoPage() {
+    async function buscarProfessor() {
+        const email = form.getValues('professores')?.trim()
+        if (!email || email.length < 5) {
+            toast.warn('Informe um email válido.')
+            return
+        }
+
+        try {
+            const resultado = await UsuarioService.buscarPorEmail(email)
+            if (resultado) {
+                setProfessores(resultado)
+                form.setValue('professores', resultado.nome) // Exibe nome ao invés do email
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)